packages/cactus-plugin-ledger-connector-iroha/src/main/json/openapi.json

```diff
@@ -173,9 +173,646 @@
+          },
+          "amount": {
+            "type": "number"
+          }
+        }
+      },
+      "TransferAssetRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Transfer Asset.",
+        "type": "object",
+        "required": [
+          "srcAccountId",
+          "destAccountId",
+          "assetId",
+          "description",
+          "amount"
+        ],
+        "properties": {
+          "srcAccountId": {
+            "type": "string"
+          },
+          "destAccountId": {
+            "type": "string"
+          },
+          "assetId": {
+            "type": "string"
+          },
+          "description": {
+            "type": "string"
+          },
+          "amount": {
+            "type": "number"
+          }
+        }
+      },
+      "GetSignatoriesRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Get Signatories.",
+        "type": "object",
+        "required": [
+          "accountId"
+        ],
+        "properties": {
+          "accountId": {
+            "type": "string"
+          }
+        }
+      },
+      "GetAccountRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Get Account.",
+        "type": "object",
+        "required": [
+          "accountId"
+        ],
+        "properties": {
+          "accountId": {
+            "type": "string"
+          }
+        }
+      },
+      "GetAccountDetailRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Get Account Detail.",
+        "type": "object",
+        "required": [
+          "accountId",
+          "key",
+          "writer",
+          "pageSize",
+          "paginationKey",
+          "paginationWriter"
+        ],
+        "properties": {
+          "accountId": {
+            "type": "string"
+          },
+          "key": {
+            "type": "string"
+          },
+          "writer": {
+            "type": "string"
+          },
+          "pageSize": {
+            "type": "number"
+          },
+          "paginationKey": {
+            "type": "string"
+          },
+          "paginationWriter": {
+            "type": "string"
+          }
+        }
+      },
+      "GetAssetInfoRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Get Asset Info.",
+        "type": "object",
+        "required": [
+          "assetId"
+        ],
+        "properties": {
+          "assetId": {
+            "type": "string"
+          }
+        }
+      },
+      "GetAccountAssetsRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Get Account Assets.",
+        "type": "object",
+        "required": [
+          "accountId",
+          "pageSize",
+          "firstAssetId"
```

Comment:
this is optional, according to: https://iroha.readthedocs.io/en/develop/develop/api/queries.html#get-account-assets